samples/bot-types-of-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,161 @@
+---
+page_type: sample
+description: Demonstrating the feature where user can send different types of cards using bot.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "01/13/2023 05:00:17 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-types-of-cards-nodejs
+
+---
+# Types of Cards
+
+This sample shows the feature where user can send different types of cards using bot.
+
+## Interaction with app
+
+![Types Of Cards](Images/TypesOfCards.gif)
+
```

Comment:
We need to add one more section for demo manifest of C# code:
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/636/files#diff-7bb681f0cc7d221ebbf0b17dacc3a1b81bf3bb250a4770a3162eaee026e62e57